pywinauto/UIAElementInfo.py

```diff
@@ -167,29 +167,46 @@ def parent(self):
         else:
             return None
 
-    @property
-    def children(self):
-        "Return list of immediate children for the element"
-        children = []
-        
-        childrenArray = self._element.FindAll(_tree_scope['children'], _true_condition)
-        for childNumber in range(childrenArray.Length):
-            childElement = childrenArray.GetElement(childNumber)
-            children.append(UIAElementInfo(childElement))
+    def _get_elements(self, scope, cond=_true_condition):
+        "Find all elements according to the given scope and conditions"
+        elements = []
+        ptrs_array = self._element.FindAll(scope, cond)
+        for num in range(ptrs_array.Length):
+            child = ptrs_array.GetElement(num)
+            elements.append(UIAElementInfo(child))
+
+        return elements
+
+    def _build_condition(self, process = None, class_name = None, title = None):
```

Comment:
Often QC finds false alarms as "new issues" but now I agree with QC. `self` is not used here so it might be function or static method. As far as I remember, pseudo-private methods should start with `__` (double underscore). Single one is used for functions.